.github/CONTRIBUTING.rst

```diff
@@ -105,6 +105,8 @@ Here's how to make a one-off code change.
 
    - Please ensure that the code you write is well-tested.
 
+        - In addition to that, we provide the `dev` marker for pytest. If you write one or multiple tests and only want to run those, you can decorate them via `@pytest.mark.dev` and then run it with minimal overhead with `pytest ./path/to/test_file.py -m dev`.
```

Comment:
> `pytest ./path/to/test_file.py -m dev`

We don't need to specify the path, just the marker itself is fine

Reply:
yes, thats why I wrote minimal overhead. Specifying the file will decrease the amount of unnecessary tests pytest collects, which can be a lot in this project.